templates/templates/_navigation-support-h.html

```diff
@@ -0,0 +1,184 @@
+              MAAS support
+            </h4>
+            <p class="p-p--small">
+              Canonical offers Standard and Advanced support plans priced per-machine or, for large volume per-region pricing.
+            </p>
+            <p class="p-heading--five">From $5 per month *</p>
+            <a href="/support/plans-and-pricing#maas" class="p-button--small p-button--positive u-align-text--left">
+              Get MAAS support
+            </a>
+          </div>
+          <h4 class="u-show--small p-muted-heading u-hide--medium u-hide--large">
+            MAAS support
+          </h4>
+          <ul class="p-text-list--small is-bordered">
+            <li class="p-list__item u-show--small u-hide--medium u-hide--large"><a href="/support/plans-and-pricing#maas">Get MAAS support</a></li>
+            <li class="p-list__item">Ubuntu and CentOS deployment</li>
+            <li class="p-list__item">Windows, RHEL, and custom image creation and deployment</li>
+            <li class="p-list__item">High availability (HA) support</li>
+            <li class="p-list__item">Extended Security Maintenance</li>
+            <li class="p-list__item">Livepatch: automatic kernel security hotfixes</li>
+            <li class="p-list__item">Landscape: the systems management tool</li>
+            <li class="p-list__item">Knowledge Base</li>
+            <li class="p-list__item">Support: phone and web</li>
+          </ul>
+        </div>
+        <!-- Ubuntu Advantage -->
+        <div class="tablet-col-2 col-3 p-card--navigation">
+          <div class="u-hide--small">
+            <h4 class="p-heading-four is-dense">
+              Ubuntu Advantage
+            </h4>
+            <p class="p-p--small">
+              Ubuntu Advantage is the professional package of tools, technology and expertise for servers and desktops
+            </p>
+            <p class="p-heading--five">From $75 per year *</p>
+            <a href="/support/plans-and-pricing#ua-support" class="p-button--small p-button--positive u-align-text--left">
+              Get Ubuntu Advantage
+            </a>
+          </div>
+          <h4 class="u-show--small p-muted-heading u-hide--medium u-hide--large">
+            MAAS support
+          </h4>
+          <ul class="p-text-list--small is-bordered">
+            <li class="p-list__item u-show--small u-hide--medium u-hide--large"><a href="/support/plans-and-pricing#ua-support">Get Ubuntu Advantage</a></li>
+            <li class="p-list__item">Extended Security Maintenance</li>
+            <li class="p-list__item">Livepatch: automatic kernel security hotfixes</li>
+            <li class="p-list__item">Landscape: the systems management tool</li>
+            <li class="p-list__item">Knowledge Base</li>
+            <li class="p-list__item">Support: phone and web</li>
+          </ul>
+        </div>
+      </div>
+      <div class="row">
+        <p class="p-heading--six-original">* Minimums apply</p>
```

Comment:
Could you just use `<small><i>* Minimums apply<I><small>` save us another snowflake

Reply:
Done